optuna_dashboard/artifact/_backend.py

```diff
@@ -170,31 +191,49 @@ def _artifact_prefix(trial_id: int) -> str:
     return ARTIFACTS_ATTR_PREFIX + f"{trial_id}:"
 
 
-def _get_artifact_meta(
+def get_artifact_meta(
```

Comment:
[question] should we return both dashboard_artifact_meta and optuna_artifact_meta similart to `list_trial_artifacts`?

Reply:
No. When retrieving meta information from an artifact_id, it will exist in either dashboard_artifact_meta or optuna_artifact_meta, but not in both.